browser/src/map/Clipboard.js

```diff
@@ -877,8 +851,10 @@ L.Clipboard = L.Class.extend({
 	},
 
 	_asyncAttemptNavigatorClipboardWrite: async function() {
-		const command = this._unoCommandForCopyCutPaste;
-		await this._sendCommandAndWaitForCompletion(command);
+		if (this._commandCompletion)
+			throw 'Already have ' + this._commandCompletion.length + ' pending clipboard command(s)';
+		else
+			this._commandCompletion = true;
```

Comment:
do you ever send the command here? It looks like you remove `_sendCommandAndWaitForCompletion`, possibly without realizing that I moved the `app.socket.sendMessage` line into it...

Reply:
From your PR description you mention
> _doCopyCut function is already sending the relevant command

but I think the function is also run from `_execCopyCutPaste`, where I think the command is only run here... so I guess this eliminates a duplicate for the _doCopyCut case but I'm worried it might break the _execCopyCutPaste case in the process